xicam/plugins/guiplugin.py

```diff
@@ -19,6 +18,7 @@ class GUIPlugin(PluginType):
 
     is_singleton = True
     needs_qt = True
+    supports_ensembles = False
```

Comment:
also hold off on these for more discussion

Reply:
Removed these changes to GUIPlugin, should be back to original state in `main`. I kept SAXS/XPCS inheriting the `EnsembleGUIPlugin`, I think that should be okay?